benchmarks/mysql-kafka-cdc/conduit/pipeline.yml

```diff
@@ -0,0 +1,24 @@
+version: "2.2"
+pipelines:
+  - id: mysql-to-kafka
+    status: stopped
+    name: mysql-to-kafka
+    description: MySQL to Kafka
+    connectors:
+      - id: mysql-source
+        type: source
+        plugin: builtin:mysql
+        settings:
+          dsn: root:root@tcp(mysql1:3306)/conduit
+          tables: users
+          sdk.batch.size: 10000
```

Comment:
WDYT about disabling the schemas here too (if they are disabled in the Debezium connector as well)?